tests/testthat/helper-test-check.R

```diff
@@ -26,6 +26,43 @@ check_graph <- function(p, filename, max_distortion = 0.99) {
   }
 }
 
+check_gif <- function(gif, filename, max_distortion = 0.99) {
```

Comment:
The current travis failure is spurious - it's just an artifact of how the code checks that there aren't any unused test reference files. Changing check_gif to accept the list of gg, rather than a filepath, and then saving the gif into the tempdir() (as is done by check_graph) _should_ (I hope) get around this.
```
check_gif <- function(gg_list, filename, max_distortion = 0.99) {
comp_location <- paste0(tempdir(), "/", filename, ".gif")
  agg_slides(gg_list, comp_location)
```